docs.md

```diff
@@ -171,3 +183,232 @@ As you can tell after running the code and logging out the method, we get the fo
+//       trust: '0.00'
+//     },
+//     {
+//       id: '2682963f-06b1-47d7-a2e1-1f8ec6ae98dc',
+//       created_date: '2020-10-08T02:39:44.071810Z',
+//       modified_date: '2020-10-08T02:39:44.071853Z',
+//       account_number: 'cc8fb4ebbd2b9a98a767e801ac2b0d296ced88b5d3b7d6d6e12e1d2d7635d724',
+//       trust: '0.00'
+//     },
+//    ....18 more accounts
+//   ]
+// }
+```
+
+#### Options Object
+
+As you can see in the previous section, `bank.getAccount` returns a bunch of accounts with some extra data like count, next, and previous. This is where the options object plays in.
+
+Since we didn't specify the options object at all in this code, the default one - `{ defaultPagination: { limit: 20, offset: 0 } }` is used.
+
+However, if we do pass in our own options object, we can customize the `offset` and `limit` for each api call.
+
+```js
+const bank = new Bank("http://143.110.137.54", { defaultPagination: { limit: 2, offset: 0 } });
+bank.getAccounts();
+// {
+//   count: 98,
+//   next: 'http://143.110.137.54/accounts?limit=20&offset=2',
+//   previous: null,
+//   results: [
+//     {
+//       id: '5a8c7990-393a-4299-ae92-2f096a2c7f43',
+//       created_date: '2020-10-08T02:18:07.346849Z',
+//       modified_date: '2020-10-08T02:18:07.346914Z',
+//       account_number: 'a37e2836805975f334108b55523634c995bd2a4db610062f404510617e83126f',
+//       trust: '0.00'
+//     },
+//     {
+//       id: '2682963f-06b1-47d7-a2e1-1f8ec6ae98dc',
+//       created_date: '2020-10-08T02:39:44.071810Z',
+//       modified_date: '2020-10-08T02:39:44.071853Z',
+//       account_number: 'cc8fb4ebbd2b9a98a767e801ac2b0d296ced88b5d3b7d6d6e12e1d2d7635d724',
+//       trust: '0.00'
+//     },
+//    ....only 2 accounts
+//   ]
+// }
+```
+
+As expected, we can also pass in the options object in the api call itself.
+
+```js
+bank.getAccounts({ limit: 10, offset: 30 });
+```
+
+> The options object passed into the api call has higher precedence than the one passed into the bank constructor.
+
+> The behaviour of the options object is similar with all api calls.
+
+#### Updating Accounts
+
+This is an api call to update the level of trust of a specific account.
+
+We use `Bank.updateAccount` for this.
+
+```js
+const account = new Account("0cdd4ba04456ca169baca3d66eace869520c62fe84421329086e03d91a68acdb", "fakeSigningKeyHex");
+bank.updateAccount("0cdd4ba04456ca169baca3d66eace869520c62fe84421329086e03d91a68acdb", 32, account); // 32 is the amount of trust
+// {
+//   "id": "64426fc5-b3ac-42fb-b75b-d5ccfcdc6872",
+//   "created_date": "2020-07-14T02:59:22.204580Z",
+//   "modified_date": "2020-07-21T00:58:01.013685Z",
+//   "account_number": "0cdd4ba04456ca169baca3d66eace869520c62fe84421329086e03d91a68acdb",
+//   "trust": "32"
+// }
+```
+
+Now the trust of the account supplied would be 32 instead of whatever it was before.
+
+#### Getting Transactions
+
+Pretty similar to [Getting Accounts](#getting-accounts).
```

Comment:
It might be better to say "Getting a list of transactions is fairly similar to [...]".